python/hail/api1/context.py

```diff
@@ -843,16 +843,30 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, min_p
     @handle_py4j
     @record_method
     @typecheck_method(path=oneof(strlike, listof(strlike)),
+                      key_expr=strlike,
+                      annotation_types=listof(Type),
+                      number_of_annotations=integral,
```

Comment:
Do you want to just get this from len(annotation_types)?

Reply:
yeah, that's what I was thinking. is that ok?

Reply:
Yes, please!